helm/helm.bzl

```diff
@@ -3,7 +3,7 @@
 load("//helm/private:helm_install.bzl", "helm_install", "helm_push", "helm_reinstall", "helm_uninstall")
 load("//helm/private:helm_package.bzl", "helm_package")
 
-def helm_chart(name, images = [], deps = None, tags = [], install_name = None, **kwargs):
+def helm_chart(name, oci_images = [], deps = None, tags = [], install_name = None, **kwargs):
```

Comment:
I don't think you need to change the parameter name here, can you leave the interface the same so others don't need to make as many changes when udpating?

Reply:
this is actually something I can do post-merge